Episode_of_care.sas

```diff
@@ -58,36 +87,57 @@ drop EoC:;
 run;
 
 /*
-Nye tidsvariabler, b�de dato og tidspunkt p� d�gnet for ut og inn
+Nye tidsvariabler, b�de dato og tidspunkt p� d�gnet for ut og inn.
+Definerer ogs� tmp_poli, som er lik 1 hvis aktivitetskategori3 ikke er (1,2)
 */
 data &dsn;
 set &dsn;
+tmp_poli = .;

```

Comment:
Define `tmp_poli` so I can sort on it later

Reply:
Det er lurt!  :-)